DatadogObjc/Sources/RUM/RUM+objc.swift

```diff
@@ -649,17 +649,30 @@ public class DDRUMMonitor: NSObject {
 
     @objc
     public func addAttribute(
-        forKey key: String,
+        forKey key: AttributeKey,
```

Comment:
/suggestion Lets keep it as `String`, I'm not 100% this will be properly exposed to objc. Plus, we use `String` for all `attributes`.